server/src/routes/api/v2/wallets/transfers.js

```diff
@@ -29,12 +29,23 @@ const withJobs = async (transferEvents) => {
   return jobsByTxHashes
 }
 
-const getWalletJobs = (walletAddress, tokenAddress) => QueueJob.find({
-  'data.walletAddress': walletAddress,
-  'data.transactionBody': { '$exists': true },
-  'data.transactionBody.status': 'pending',
-  'data.transactionBody.tokenAddress': tokenAddress.toLowerCase()
-})
+const getWalletJobs = async (walletAddress, tokenAddress, blockNumber) => {
+  const pendingJobs = await QueueJob.find({
+    'data.walletAddress': walletAddress,
+    'data.transactionBody': { '$exists': true },
+    'data.transactionBody.status': 'pending',
+    'data.transactionBody.tokenAddress': tokenAddress.toLowerCase()
+  })
+  const fiatProcessingJobs = await QueueJob.find({
+    'name': 'fiat-processing',
```

Comment:
I think the name is `fake-deposit`,
need to search for `data. actionType` instead